### PR TITLE
Add additional CODEOWNERS

### PR DESCRIPTION
**Pull request by wborn**
Here are some more entries for the CODEOWNERS file mainly based on initial contributions.

### CODE REVIEW

**Comment by davidgraeff** [comment]
Guys. Let's merge that in and add further changes in later PRs :)

File: CODEOWNERS [hunk 1]
```diff
@@ -5,9 +5,163 @@
...
+/addons/binding/org.openhab.binding.sensebox/ @hakan42
+/addons/binding/org.openhab.binding.silvercrestwifisocket/ @jmvaz
+/addons/binding/org.openhab.binding.sleepiq/ @syphr42
+/addons/binding/org.openhab.binding.smaenergymeter/ @monnimeter
+/addons/binding/org.openhab.binding.smartmeter/ @msteigenberger
+/addons/binding/org.openhab.binding.smartmeter.test/ @msteigenberger
+/addons/binding/org.openhab.binding.solaredge/ @alexf2015
+/addons/binding/org.openhab.binding.solarlog/ @johannrichard
+/addons/binding/org.openhab.binding.somfytahoma/ @octa22
+/addons/binding/org.openhab.binding.squeezebox/ @digitaldan
```
Comment (mhilbush) [inline CODEOWNERS]: You can add me for squeezebox, as I've been doing most of the changes lately.

**Comment by wborn** [comment]
If my memory serves me well maybe:
* @davidgraeff should be added to hueemulation?
* @martinvw should be added to rfxcom?
* @jarlebh no longer uses/maintains the tellstick?

File: CODEOWNERS [hunk 2]
```diff
@@ -5,9 +5,163 @@
...
+/addons/binding/org.openhab.binding.evohome/ @Nebula83
+/addons/binding/org.openhab.binding.exec/ @kgoderis
+/addons/binding/org.openhab.binding.feed/ @svilenvul
+/addons/binding/org.openhab.binding.feed.test/ @svilenvul
+/addons/binding/org.openhab.binding.feican/ @Hilbrand
+/addons/binding/org.openhab.binding.folding/ @fa2k
+/addons/binding/org.openhab.binding.freebox/ @clinique
+/addons/binding/org.openhab.binding.fronius/ @trokohl
+/addons/binding/org.openhab.binding.ftpupload/ @paulianttila
+/addons/binding/org.openhab.binding.gardena/ @paulianttila
```
Comment (paulianttila) [inline CODEOWNERS]: Gardena binding owner should be @gerrieg not me?

Comment (wborn) [reply CODEOWNERS]: I'll correct that! Copy/paste is always an issue on Linux desktops. :wink:

**Pull request merged**